jsx-templates/button.jsx

```diff
@@ -0,0 +1,29 @@
+//this template will live on USWDS repo
+//--sandbox for styles.css--//
+//https://codesandbox.io/s/bold-voice-xwend?file=/src/App.js
+
```

Comment:
In the comments, put info about what gets passed into `<button>` and where it comes from